java/ql/src/semmle/code/java/frameworks/apache/Collections.qll

```diff
@@ -0,0 +1,264 @@
+/** Definitions related to the Apache Commons Collections library. */
+
+import java
+private import semmle.code.java.dataflow.FlowSteps
+private import semmle.code.java.dataflow.ExternalFlow
```

Comment:
This should be a bidirectional import (i.e. this file should be imported in `ExternalFlow.qll`). Right know I believe this only (accidentally) works due to the import in `NullGuards.qll`.